app/discovery/events.go

```diff
@@ -140,6 +142,10 @@ func (e *EventNotif) group(image string) string {
 }
 
 func (e *EventNotif) isAllowed(containerName string) bool {
+	if e.includesPattern != "" {
+		re, _ := regexp.Compile(e.includesPattern)
```

Comment:
I'd suggest to move it to the constructor and not to compile on each call

Reply:
Fair point, will do.

Reply:
@umputun Branch updated.